File: RoundTable SDK for DotNET/src/interfaces/INamedFieldTemplateManager.cs
using System;

namespace RoundTable.ProjectServer.Services.NamedFieldTemplateManager
{
	public interface INamedFieldTemplateManager
	{
		int addNamedFieldTemplate(Project project, Object typeContext, NamedFieldTemplate template);
   
		void dropNamedFieldTemplate(int namedFieldTemplateId);
   
		void modifyNamedFieldTemplate(NamedFieldTemplate template);
   
		NamedFieldTemplate getNamedFieldTemplate(int namedFieldTemplateId);
   
		NamedFieldTemplate[] getAllNamedFieldTemplatesForProjectAndType(Project project, Object typeContext);
	}
}

File: RoundTable Project Server/src/com/dommoni/roundtable/projectServer/services/namedFieldTemplateManager/INamedFieldTemplateManager.java
package com.dommoni.roundtable.projectServer.services.namedFieldTemplateManager;

import com.dommoni.roundtable.NamedFieldTemplate;
import com.dommoni.roundtable.Project;

public interface INamedFieldTemplateManager
{
   public int addNamedFieldTemplate(Project project, Object typeContext, NamedFieldTemplate template)
      throws Exception; 
   
   public void dropNamedFieldTemplate(int namedFieldTemplateId)
      throws Exception;
   
   public void modifyNamedFieldTemplate(NamedFieldTemplate template)
      throws Exception;
   
   public NamedFieldTemplate getNamedFieldTemplate(int namedFieldTemplateId)
      throws Exception;
   
   public NamedFieldTemplate[] getAllNamedFieldTemplatesForProjectAndType(Project project, Object typeContext)
      throws Exception;
}
/*
 *
 * $Log: INamedFieldTemplateManager.java,v $
 * Revision 1.1  2005/10/17 07:45:59  brian
 * Initial Revision.
 *
 *
 */